### ISSUE TITLE
We would love to adopt this package!

### ISSUE THREAD

**Issue by yyx990803** [liximomo/vue-function-api#14]
Hi @liximomo - thank you very much for working on this package, something we wish we had when the RFC thread blew up.

We are considering adopting this package to serve as the official 2.x function API plugin, and in the process you are welcome to join the core team. There are still a few changes to the RFC we are considering, and would love to work with you to see that reflected in this package. Would you be interested?

**Comment by liximomo** [comment]
That would be great! I'm willing to contribute to the vuejs community.

**Comment by liximomo** [comment]
@yyx990803 Should I port this repo to vuejs org.

**Comment by yyx990803** [comment]
@liximomo yes, I think you have the privilege to do it. Just let me know if it doesn't work.

**Comment by edimitchel** [comment]
Transfered here : https://github.com/vuejs/vue-function-api
